amplify/backend/function/risestandstrongcb34053a/src/routes/shifts.js

```diff
@@ -198,7 +199,18 @@ router.get('/:startTimestamp', async (req, res) => {
  *            description: startTimestamp not found
  */
 router.put('/:startTimestamp', async (req, res) => {
-   res.end();
+   let startTimestamp = req.params.startTimestamp;
+   let body = {
+      startTimestamp: parseInt(startTimestamp),
+      ...req.body
+   }
+   try {
+     const shift = await putShift(body);
+     res.send(shift)
```

Comment:
You shouldn't send the shift in the response just return a 200 because this is a PUT not a GET